doc/api/readline.md

```diff
@@ -703,7 +703,16 @@ added: v17.0.0
 The `readlinePromises.createInterface()` method creates a new `readlinePromises.Interface`
 instance.
 
-```js
+```mjs
+import { createInterface } from 'node:readline/promises';
+import { stdin, stdout } from 'node:process';
+const rl = createInterface({
+  input: stdin,
+  output: stdout,
+});
+```
+
+```cjs
 const readlinePromises = require('node:readline/promises');
```

Comment:
If you use destructuring in the ESM, it should probably also be used here.

Reply:
Ok! I generally try not to touch the examples that are already there, but if you think that'd be better I'll do it!

Should I do it for that example in particular or for the rest of the examples as well?

Thanks for the feedback! :raised_hands:

Reply:
IMO all of them

Reply:
Done!

You were right, now it looks more concise with [the first example](https://nodejs.org/api/readline.html#readline) as well.

Thank you! :blush: